.travis.yml

```diff
@@ -1,6 +1,4 @@
 language: clojure
-lein: lein2
-script: lein2 with-profile dev:dev,1.4:dev,1.5:dev,1.6:dev,1.7 test
+script: lein test
 jdk:
-  - openjdk7
-  - openjdk6
+  - openjdk8
```

Comment:
Why is JDK8 necessary?

Reply:
Relevant closure compiler requires Java 8. It depends on which version of clojurescript we use. Generally, clout does not depend on specific clojurescript version.

Reply:
Tested agains Java 6, 7, 8

Reply:
Relevant closure compiler requires Java 8.